librz/include/rz_endian.h

```diff
@@ -65,6 +65,23 @@ static inline void rz_write_at_be16(void *dest, ut16 val, size_t offset) {
 	rz_write_be16(d, val);
 }
 
+static inline ut32 rz_read_be24(const void *src) {
+	const ut8 *up = (const ut8 *)src + 2;
+	return rz_read_be16(src) | rz_read_be8(up);
```

Comment:
I'm not sure about this...

It should probably be something like:
```
rz_read_be8(src) << 16 | rz_read_be8(src+1) << 8 | rz_read_be8(src+2)
```
no?

Reply:
You are right, yes.